js/even-numbers.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+// getting user value
+const maxValue = Number(prompt("Enter your number: "));
+
+// with for:
+if(isNaN(maxValue)){
+    console.log("Error: not a number!");
+} else {
+    const isPositive = maxValue > 0 ? 1 : -1;
+    for(let i = 0; i < Math.abs(maxValue); i++){
+        if(i % 2 === 0 && i !== 0){
```

Comment:
```suggestion
    for(let i = 1; i < Math.abs(maxValue); i++){
        if(i % 2 === 0){
```

...і буде те ж саме, але простіше, а можна зробити ще простіше і зрозуміліше, рекомендую подумати.